src/components/item/item-reorder.ts

```diff
@@ -142,6 +142,10 @@ export class ReorderIndexes {
   host: {
     '[class.reorder-enabled]': '_enableReorder',
     '[class.reorder-visible]': '_visibleReorder',
+    '[class.reorder-side-left]': 'side === "left"',
+    '[class.reorder-side-right]': 'side === "right"',
+    '[class.reorder-side-start]': 'side === "start"',
+    '[class.reorder-side-end]': 'side === "end"'
```

Comment:
@manucorporat what would be the correct way here to add the classes? Should I can just use `[ngClass]` with a string.